docs/BUILD.md

```diff
@@ -98,16 +137,12 @@ s2n-tls has a dependency on a libcrypto library. A supported libcrypto must be l
   - OCSP features are not supported with BoringSSL.
 - [LibreSSL](https://www.libressl.org/)
 
-By default, s2n-tls will attempt to find a system libcrypto to link with when building. However, this search can be overridden to any of the above libcryptos by specifying the install directory with the `CMAKE_PREFIX_PATH` flag.
-
-For help building a desired libcrypto on your platform, please consult the project's build documentation.
+For help building a desired libcrypto on your platform, consult the build documentation for that libcrypto.
 
 ### AWS-LC
 
 [AWS-LC](https://github.com/aws/aws-lc) is the recommended libcrypto to use with s2n-tls due to increased performance and security. See the [AWS-LC build documentation](https://github.com/aws/aws-lc/blob/main/BUILDING.md) for information on building AWS-LC.
 
-The `CMAKE_INSTALL_PREFIX` option can be provided when building AWS-LC to specify where AWS-LC will be installed. The install path for AWS-LC should be provided when building s2n-tls, via the `CMAKE_PREFIX_PATH` option. This will ensure that s2n-tls is able to find the AWS-LC library artifact to link with.
-
```

Comment:
Nit: I'm guessing you removed this because aws-lc should own its own build instructions. But since we're specifically calling out aws-lc as our recommended option, maybe we want to keep these quick / easy access instructions on building the two libraries together.